dataactvalidator/config/programActivityFields.csv

```diff
@@ -13,7 +13,7 @@ grossoutlaysdeliveredorderspaidtotal_fyb,gross_outlays_delivered_or_fyb,TRUE,flo
 grossoutlaysundeliveredordersprepaidtotal_cpe,gross_outlays_undelivered_cpe,TRUE,float,21,"B5, B6, A18",
 grossoutlaysundeliveredordersprepaidtotal_fyb,gross_outlays_undelivered_fyb,TRUE,float,21,"A16, B5, B6",
 mainaccountcode,main_account_code,TRUE,str,4,A1,TRUE
-objectclass,object_class,TRUE,str,4,B11,
+objectclass,object_class,TRUE,str,4,B11,TRUE
```

Comment:
You probably don't want to add the "padded_flag" field here. For some reason the field_length is 4 (still not sure why since they all seem to be 3 long, but you can ask someone that knows more about the rules than me about that). This means that every single objectclass field will be padded to 4 long with a leading 0. You should double-check that with several values just to be sure.